libros/packages/print_options_list.py

```diff
@@ -8,18 +13,19 @@
 """
 
 options = [
-    (1, 'Check out a book'),
-    (2, 'Return a book'),
-    (3, 'Add a new book to the library'),
-    (4, 'Delete a book from the library'),
+    (1, 'Find a book'),
```

Comment:
These could probably just be strings here since they're only used below and you could avoid the cast on line 28